Maintainers_Guide.md

```diff
@@ -0,0 +1,90 @@
+on “The Role of a Maintainer” by Matthew Rocklin,
+which is a recommended reading in conjunction with this guide.
+
+## Why become a maintainer?
+
+As a BIDS maintainer you may get the chance to:
+
+* Learn to work as a team
+* Bring your expertise to the BIDS maintainers group and cover technical blind spot it may have
+* Improving your technical writing skills (for example documentation)
+* Learn to work with continuous integration and development
+* Advise and participate in the development of BIDS extensions that are most commonly associated with a publication
+
+## Responsibilities
+
+* Maintainers need to be loosely aware of the entire project
+  and use their knowledge to facilitate and initiate interactions
+  between different nodes of the project
+  and determine a reasonable and timely order for features to be added and issues to be resolved.
+* Maintainers direct other BIDS contributors in reviewing PRs,
+  writing clarifications to the specification, or other contributions.
+  Maintainers “dispatch” work teams.
+* Maintainers ensure that all contributions maintain a friendly and welcoming tone
+  to encourage productive conversations.
+* If no work team is suitable or available,
+  the final responsibility of getting the work done lies with the maintainers.
+* The development of each BIDS extension proposal should be “followed”
+  by at least one maintainer who acts as a preferential point of contact
+  between the BIDS maintainers and the BEP leads.
+
+Apart from these abstract and general responsibilities,
+maintainers within the BIDS community also need to ensure that the following tasks get done:
+
+* Keeping the [Contributors wiki](https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors) up to date
+  and assisting new contributors with adding their credits
+  and performing community inquiries to ensure contributors are credited in the [Contributors appendix](https://bids-specification.readthedocs.io/en/stable/appendices/contributors.html)
+    * Deciding what constitutes a contribution worth adding to the “Contributors list”
+* Preparing a monthly report to the BIDS Steering Group.
+  The monthly report is in the form of milestones, issues addressed,
+  and open issues raised in the past month and goals/plans for the next month.
+  The BIDS Steering Group may ask for additional information or propose a meeting to further discuss report items.
+  The report format and meetings are at the discretion of the BIDS Steering Group.
+
+Maintainers are not expected to individually be responsible for all the responsibilities listed.
+Rather, the responsibilities are distributed amongst the entire group.
+
+## Organization
+
+* The group of maintainers are a group of people with the above mentioned responsibilities,
+  who commit to convene bi-weekly meetings to discuss the project.
+* One lead maintainer represents the group to other BIDS Groups, mediates disagreement among members,
```

Comment:
I didn't know about the "lead maintainer" role. Who is it currently? Should that be documented somewhere?

Reply:
Mentioned here

https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md#maintainers-group

Reply:
I gave this a section in this document with the most recent commit now. In the future the lead maintainer role will rotate among the maintainers.